changelog/7307.trivial.rst

```diff
@@ -0,0 +1,11 @@
+:meth:`sunpy.net.jsoc.JSOCClient.fetch` called `drms` API that passed a ``progress`` keyword which added extra print statements to the console.
+This has been removed in `drms` 0.7.0, which had breaking API changes within this release.
+As a result, we increased the minimum required version of `drms` to 0.7.0.
+
+This specifically refers to the following information that was printed to the console:
+
+``"Export request pending. [id=X, status=X]"``
+``"Waiting for X seconds..."``
+``"Request not found on server, X retries left."``
+
+These were handled by `drms` and are now logging messages.
```

Comment:
It's not clear if these messages are now shown by default or not? And if not, could you add a quick explanation (with code) on how to turn them back on? (or say that it's not possible to see them any more if it isn't possible)

Reply:
Added more text and code example. Hopefully that is good enough.

Reply:
👍 liking the code example